mep/accounts/tests/test_accounts_models.py

```diff
@@ -19,10 +19,20 @@ class TestAccount(TestCase):
 
     def test_repr(self):
         account = Account()
-        # Using self.__dict__ so relying on method being correct
-        # Testing for form of "<Account {"k":v, ...}>""
-        overall = re.compile(r"<Account \{.+\}>")
-        assert re.search(overall, repr(account))
+        # unsaved
+        assert repr(account) == '<Account pk:??>'
+        # saved but no names
+        account.save()
+        assert repr(account) == '<Account pk:%s>' % account.pk
+        # one name
+        pers1 = Person.objects.create(name='Mlle Foo')
+        account.persons.add(pers1)
+        assert repr(account) == '<Account pk:%s %s>' % (account.pk, str(pers1))
```

Comment:
i wonder if just writing out the string is better? it makes the tests easier to read and understand, and also doesn't implicitly test things like `str()` along with the method you're trying to test.

doesn't really apply for pks, but for cases like here where you're passing a name it would be cool to at least see that spelled out, e.g. `'<Account pk:%s Mlle Foo>' % account.pk`

Reply:
Fair point! That would be much more readable and isolates the logic more

Reply:
I looked, and I think there's only one I can really change - and they are actually dependent on `str` functionality. I'm going to leave it.